docs/settingUp.md

```diff
@@ -59,6 +59,7 @@ More information can be found at [this documentation](https://help.github.com/ar
    * `gradle.properties`<br>
       If you want to use a JDK other than the one specified in your PATH variable, add the value to the variable `org.gradle.java.home`.
       This value must be a valid **JDK 1.7** directory.
+      **Windows users** must be careful while entering in the address. Use a **forward slash(/)** instead of the windows default backward slash(/) while entering the path to the directory. This confusion usually occurs when users copy and paste the path from Windows explorer to gradle.properties. 
```

Comment:
Replace the first sentence with '**Windows users** should use a `<second sentence here>`...'
* **forward slash(/)** -> **forward slash(`/`)**
* backward slash(/) -> **backward slash(`\`)**
* while entering the path to the directory -> when specifying the path

The third sentence is not really unnecessary.